url.bs

```diff
@@ -1690,12 +1689,8 @@ string <var>input</var>, optionally with a <a>base URL</a> <var>base</var>, opti
  <li><p>Let <var>state</var> be <var>state override</var>
  if given, or <a>scheme start state</a> otherwise.
 
- <li><p>If <var>base</var> is not given, set it to null.
-
- <li><p>Let <var>encoding</var> be <a>UTF-8</a>.
-
- <li><p>If <var>encoding override</var> is given, set <var>encoding</var> to the result of
- <a lt="get an output encoding">getting an output encoding</a> from <var>encoding override</var>.
+ <li><p>If <var>encoding</var> is not <a>UTF-8</a>, then set <var>encoding</var> to the result of
+ <a>getting an output encoding</a> from <var>encoding</var>.
```

Comment:
I did the "not UTF-8" check to avoid having to read that algorithm for UTF-8 scenarios, but maybe that's not worth it?

Reply:
Yeah, I think that adds a bit more confusion than it saves